.github/workflows/upload_app_store.yml

```diff
@@ -0,0 +1,66 @@
+name: Upload IPA to App Store
```

Comment:
This workflow describes the basic flow of uploading to App Store, we should discuss if there is anything else needed to add to this workflow.